### PR TITLE
ECMS-7563 MissingResourceException: Cant find resource for bundle org.gatein.common.i18n.ParentChildResourceBundle, key Drives.label.Collaboration

### PR DESCRIPTION
**Pull request by nttuyen**
- Add missing resource bundle
- Fix display driver name as {0} Document

### CODE REVIEW

File: packaging/wcm/webapp/src/main/resources/locale/wcm/webui_en.xml [hunk 1]
```diff
@@ -38,6 +38,8 @@
       <SharedSite>Shared Site</SharedSite>
       <PersonalDocuments>Personal Documents</PersonalDocuments>
       <UserDocuments>{0} Documents</UserDocuments>
+      <Collaboration>Collaboration</Collaboration>
+      <Trash>Trash</Trash>
```
Comment (fmaatoug) [inline packaging/wcm/webapp/src/main/resources/locale/wcm/webui_en.xml]: Updated Pr to include the Trash drive

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/sidebar/UITreeExplorer.java [hunk 2]
```diff
@@ -159,19 +163,48 @@ public String getLabel()  {
...
+                    repoService.getCurrentRepository().getConfiguration().getDefaultWorkspaceName(),
+                    repoService.getCurrentRepository()).getItem(path);
+            return groupNode.getProperty(NodetypeConstant.EXO_LABEL).getString();
+          } catch(Exception e) {
+            return id.replace(".", " / ");
+          }
+        }
       }
+    } catch (Exception ex) {
+      LOG.warn("Can not resource string for " + driveLabelKey, ex);
```
Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/sidebar/UITreeExplorer.java]: "Can not resource string for " -> "Cannot find resource string for "

Comment (nttuyen) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/sidebar/UITreeExplorer.java]: Done

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDrivesArea.java [hunk 3]
```diff
@@ -95,8 +96,12 @@ public boolean isFirstVisit() {
   public String getLabel(String id)  {
     RequestContext context = RequestContext.getCurrentInstance();
     ResourceBundle res = context.getApplicationResourceBundle();
+    String userDisplayName = "";
+    if (ManageDriveServiceImpl.USER_DRIVE_NAME.equals(id)) {
+      userDisplayName = "Users";
```
Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDrivesArea.java]: Why "Users" ? It should be the display name of the current user instead.

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIActionBar.java [hunk 4]
```diff
@@ -418,8 +418,8 @@ public String getDriveLabel() {
         try {
           driveLabel = res.getString(driveLabelKey);
         } catch (MissingResourceException ex) {
-          LOG.error("Cannot get resource string for " + driveLabelKey + " : " + ex.getMessage(), ex);
-          driveLabel = driveLabelKey;
+          //
+          driveLabel = driveName.replace(".", "/");
```
Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIActionBar.java]: The drive "Users Document" is displayed with the label "Documents" in the button. It should display the same than in the drive selector screen.

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/sidebar/UITreeExplorer.java [hunk 5]
```diff
@@ -159,19 +163,48 @@ public String getLabel()  {
...
-    } catch (MissingResourceException ex) {
-      try {
-        RepositoryService repoService = WCMCoreUtils.getService(RepositoryService.class);
-        Node groupNode = (Node)WCMCoreUtils.getSystemSessionProvider().getSession(
-                                                                                  repoService.getCurrentRepository().getConfiguration().getDefaultWorkspaceName(),
-                                                                                  repoService.getCurrentRepository()).getItem(path);
-        return groupNode.getProperty(NodetypeConstant.EXO_LABEL).getString();
-      } catch(Exception e) {
-        return id.replace(".", " / ");
+      if (ManageDriveServiceImpl.USER_DRIVE_NAME.equals(id)) {
```
Comment (thomasdelhomenie) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/sidebar/UITreeExplorer.java]: The title of the tree explorer is "Documents" when the "User Documents" drive is selected. It is not consistent.

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIActionBar.java (hunk 4 above)
Comment (nttuyen) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/control/UIActionBar.java]: Done

File: core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/sidebar/UITreeExplorer.java (hunk 5 above)
Comment (nttuyen) [inline core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/sidebar/UITreeExplorer.java]: Done

**Pull request merged**